AppStore/Ukrainian/Description.iOS.txt

```diff
@@ -0,0 +1,29 @@
+Чи цікавились ви колись «хто ця акторка?», або можливо «коли вийшов цей фільм?», або «як називався той епізод мого улюбленого серіалу»? Може, ви знавець кінематографу і замислювались «коли вийшов цей фільм, скільки років було режисеру»?
```

Comment:
Instead of "how old was this person when this movie was released", it's more like "when this movie was released, how old was the director"

More generic alternatives sounded clunky